### PR TITLE
SVGTikz: Init at 0.2

### PR DESCRIPTION
**Pull request by P-E-Meunier**
###### Motivation for this change

SVGTIkz is a rewrite-it-in-Rust™ of svg2tikz. The goal is to support more features of SVG, make it faster, make it more customisable via "filters", and possibly cache graphics.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -8237,6 +8237,8 @@ in
 
   svg2tikz = python27Packages.svg2tikz;
 
+  svg-tikz = (callPackage ../tools/typesetting/svg2tikz {}).svg_tikz {};
```
Comment (srhb) [inline pkgs/top-level/all-packages.nix]: This path doesn't appear to exist. :)

**Review by srhb — CHANGES_REQUESTED** [review]

File: pkgs/top-level/all-packages.nix (hunk 1 above)
Comment (P-E-Meunier) [reply pkgs/top-level/all-packages.nix]: Oh right, sorry about that.

Thanks Git.

**Comment by srhb** [comment]
I notice you're the author of the tool, so... Is this discrepancy between the logical name SVGTikZ and the attribute and executable name svg-tikz intentional? :grinning:

**Review by srhb — APPROVED** [review]

**Comment by xeji** [comment]
looks like the ofborg eval got stuck somehow. 
@GrahamcOfBorg eval

**Comment by prusnak** [comment]
It seems the project got renamed to https://crates.io/crates/svg-tikz and there is a new version available (0.2.1).

I am closing this PR as it hasn't been updated for more than a year, but @P-E-Meunier you are welcome to send a new one with the updated stuff.

**Pull request closed without merging**